### DISCUSSION TITLE
Using array type field in uniforms

### DISCUSSION THREAD

**Discussion by Zetra416** [vazco/uniforms#1168] [Q&A] [answered]
Hello, im new with uniforms and need to create a jsonschemabridge with field type of array, is it possible or should i use simpleschema2bridge instead ? And if possible can anyone show me how to do it ? Thanks

**Comment by radekmie** [comment, marked as answer]
Hey @Zetra416. Of course, it is possible: just use the `array` type and the `items` subschema. [Here's an example](https://uniforms.tools/playground/#?N4IgDgTgpgzlAuIBcICCATd0YwAQAoApAZQHkA5YgYwAsoBbAQwEoQAacCAezBmVEYBXeFxiMAblGQAzRgBs4HISLGSAIlDmMAnsgCMABgMd0ASzEAjOVHQz5ikFoubk8CIKgcwWqlBpc5dCgIOwVPEGhGdFIAOzldJFkwjhhaBkZkEHx8ZlwAXgA-XGAAHRjcXCouGJh4XEYAK3F83BioAHdcVCb8YHq5OQBRCG4IGCRcNw82XEE4DVlBOXhxyfdPXABrKG12rgh0VYBtEpBBGNNpffoYU4BdXABfZgBuMoqqmrrUuiYW0vKFUmpng1gmAHIMFhYDBwWx3kD4NowFAIVwLA0oFR4HCERVIDxgvBTLAJgCgUC5OZ4GS8RTJsjUbhwYwRjpcYD6bgQQxVn0kSiIbUIKYYgBzcFPOkVR7wzlPOUyt4xBHSc7Y0zVSqReBQABq8lM6EYIgg-B-6Vy5KBn1quHEhuNppajXEADoqvQwKZrOa0kxXmU6dB4IIIOV8PQuEE5LlCsVpfbHSb9pHo5pAyr5RVLgQHVSnfs3cFRngAGRlpMFlMQYsjfYwN3WcXwGhWxMVENh8p9ILwRg-1b5o01uulp7KrmPOmPSdSrMfap2i1MA3V515bVQE365Omv2_FjKhFd8OtDq4EgUaiHgBCIvQYqgB_SMxXjDXI9NmeeOXYIBgfx2gASTiUUoGGUZQgcRgYG0GIqFiYhBAsegQWg8JYPgxCYk_J0pESewoFlEBWwYAiAPSGJiSoEBHiAA).

**Reply by Zetra416** [reply]
thanks
